Constellation Final Changes

MaxRS07 opened the pull request:
Constellation item changes
also shaders + helpers + img assets

SpectralSlime committed:
Final constellation changes

SpectralSlime committed:
forgot to re add the journal stuff

SpectralSlime committed:
jounral entry fixes

MaxRS07 committed:
Merge branch 'Tyfyter:dev' into dev

Tyfyter committed:
Merge branch 'dev' into dev

Tyfyter commented:
Could use some optimization, but looks good otherwise, could you describe the changes so I can make sure I'm not missing anything?

Tyfyter committed:
Merge branch 'dev' into dev

Tyfyter committed:
Remove unused usings

Tyfyter committed:
Optimized Constellation

The biggest culprit was initializing and setting up a new `Nodes` every time it's accessed and then again every time `NodeConnections` is accessed as part of initializing and setting that up every time it's accessed

Tyfyter committed:
Syncing

Tyfyter commented:
Last change left is just code style to the `const`s, what's `EXPNDRPR` short for? I assume `EXPND` is short for expand, but I don't have a guess for the "RPR".

Tyfyter committed:
Update Constellation.cs

Tyfyter committed:
Merge branch 'dev' into dev
